docker/ubuntu_18.04-dev.dockerfile

```diff
@@ -41,6 +40,7 @@ RUN if [ "${py_version%.?}" -eq 3 ] ; \
             update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 10; \
     fi;\
     pip install --force-reinstall \
+            cmake \
```

Comment:
we are intentionally only pip-installing python packages and instead relying on the system to install CMake